src/App.jsx

```diff
@@ -14,70 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const startIndex = goods.findIndex(elem => elem === 'Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const [value, setValue] = useState(goods[startIndex]);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {!value
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            {value}
+            {' '}
+            is selected
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            {value
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+            && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => {
+                setValue('');
+              }}
+            />
+            )
+          }
+          </h1>
+        )
+      }
+      <table className="table">
+        <tbody>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={(value === good) && 'has-background-success-light'}
```

Comment:
It is better to use libraries like classnames or clsx for conditional classnames